specification/sections/01-flag-evaluation.md

```diff
@@ -219,6 +219,20 @@ It's recommended to provide non-blocking mechanisms for flag evaluation, particu
 
 > In cases of abnormal execution, the `evaluation details` structure's `error message` field **MAY** contain a string containing additional details about the nature of the error.
 
+##### Requirement 1.4.13
+
+> If the `flag metadata` field in the `flag resolution` structure returned by the configured `provider` is set, the `evaluation details` structure's `flag metadata` field **MUST** contain that value. Otherwise, it **MUST** contain an empty record.
```

Comment:
Are you differentiating here between an empty record and `null`,  or unset?

Reply:
Yeah, essentially saying the OF SDK will do the work of converting missing (null or unset) flag metadata into an empty record, so that consumers don't have to handle that themselves.